### ISSUE TITLE
terminus background color and theme changes are not reflecting

### ISSUE THREAD

**Issue by roguexray007** [randy3k/Terminus#230]
os : mac os
sublime text version : 3
theme: guna
<img width="1440" alt="Screenshot 2020-07-16 at 5 04 44 PM" src="https://user-images.githubusercontent.com/24954929/87666634-948c7700-c786-11ea-8c12-1f1bbe468215.png">

changed "theme": "<default provided by terminus>", no change happens

**Comment by roguexray007** [comment]
no colors as in issue https://github.com/randy3k/Terminus/issues/220

**Comment by heitorsilva** [comment]
Happens to me as well...
I'm on MacOS Catalina, with Sublime Text 3, and my default shell is ZShell, with oh-my-zsh being loaded by default.

If I change the theme, nothing changes on terminus...

**Comment by randy3k** [comment]
Did you enable 256 color?

**Comment by heitorsilva** [comment]
It doesn't matter if 256 color is ON or OFF, it doesn't work either way. Example of settings:

```
{
	"256color": true,
	"default_config":
	{
		"linux": null,
		"osx": "Zsh",
		"windows": "Command Prompt"
	},
	"theme": "broadcast"
}
```
or 
```
{
	"256color": false,
	"default_config":
	{
		"linux": null,
		"osx": "Zsh",
		"windows": "Command Prompt"
	},
	"theme": "broadcast"
}
```

none work...

is there a chance of oh-my-zsh being in the way of Terminus, to apply the right theme?

my Terminus always looks like this:

<img width="568" alt="Screenshot 2020-07-17 at 15 39 35" src="https://user-images.githubusercontent.com/878699/87792124-bbf54980-c843-11ea-9034-ee4ff3d233f9.png">

**Comment by randy3k** [comment]
How about changing the `term` setting to `xterm-256color`?

**Comment by heitorsilva** [comment]
Nope... no changes. I tried ansi also, but it doesn't work.

<img width="1193" alt="Screenshot 2020-07-17 at 17 53 50" src="https://user-images.githubusercontent.com/878699/87805994-7f7f1900-c856-11ea-928d-b068c121ab4f.png">

**Comment by randy3k** [comment]
It seems that the theme has been applied. What is your issue there?

**Comment by heitorsilva** [comment]
No, the theme doesn't change... doesn't matter what I select as theme, it always stays with the appearance of the screenshot I showed you.

**Comment by randy3k** [comment]
The theme only controls ansi colors 0 to 16. Are your prompt etc using 256 color sequences?

**Comment by heitorsilva** [comment]
As you can see in the screenshot, the terminal is configured with `xterm-256colour`.

I tried with `ansi` also, but had no effect.

**Comment by randy3k** [comment]
I am lost, why are you showing macOS terminal?

**Comment by heitorsilva** [comment]
In here https://github.com/randy3k/Terminus/issues/230#issuecomment-660112835 I showed you my Terminus configs, and doesn't matter if I have 256color `true` or `false`.

When you said about `term` setting to `xterm-256color` (https://github.com/randy3k/Terminus/issues/230#issuecomment-660174734), I thought you were talking about the MacOS terminal, and not Terminus.

but now I realised you were talking about `"unix_term": "xterm-256color"`.

I changed that, and still doesn't work... now my settings are like this:

```
{
	"256color": true,
	"default_config":
	{
		"linux": null,
		"osx": "Zsh",
		"windows": "Command Prompt"
	},
	"theme": "atom",
	"unix_term": "xterm-256color"
}
```

Do I have to quit Sublime and open again to make them work after saving the preferences file? Because I'm not quitting Sublime when I change the settings... should I?

**Comment by idbentley** [comment]
fwiw - I was having the same problem, and restarting sublime after each theme change was what I needed to do.

**Comment by heitorsilva** [comment]
I can also confirm that quitting Sublime and reopening makes the theme be applied, like @idbentley said (thanks).

Problem solved for me...
🤘 🎉

**Comment by randy3k** [comment]
Normally it doesn't require a restart to change theme. Not sure why.

**Comment by thezimmee** [comment]
I'm experiencing the same issue. I can successfully apply non-user themes (after restarting Sublime), but my user theme does not apply colors from `user_theme_colors`, which is what I expected. I do have `256color` set to `true` and `unix_term` is set to `xterm-256color`, though I have tried with both of those reset to their defaults as well.

@heitorsilva Are you able to apply a user theme?

Anybody know how I can apply my user theme?

**Comment by heitorsilva** [comment]
@thezimmee no, I'm not.

I'm using the `ir-black` preset.

**Comment by randy3k** [comment]
@thezimmee Did you set `theme` to `user`?

**Comment by thezimmee** [comment]
@randy3k Yes.

**Comment by randy3k** [comment]
Could you try to regenerate the theme from the command palatte?

**Comment by thezimmee** [comment]
You mean `Terminus Utilities: Generate User Theme`? I just tried that, then restarted Sublime, still no luck...

**Comment by monkape** [comment]
> Normally it doesn't require a restart to change theme. Not sure why.

I've always had to restart for theme changes to take effect.

**Comment by Penguin98kStudio** [comment]
> Normally it doesn't require a restart to change theme. Not sure why.

Same thing happens to me on my Desktop, maybe Its related to something in sublime because on my laptop (slightly older sublime or plugin) the theme change used to be instantenous. I could see changes while scrolling though the theme list

**Comment by Argus-Kan** [comment]
It happens to me as well. I need to restart sublime every time I select a new terminus theme. I was wondering if there was a place where I could see screenshots of how all the themes are going to look like. There are so many themes it gets kinda tedious while you are trying them out, not that I'm not grateful that there are so many choices ( ๑♡⌓♡๑)/
Or maybe if there was some option or command to reload terminus without needing to restart sublime, that'd nice too.
